src/services.py

```diff
@@ -80,6 +86,11 @@ def get_user(self, id):
         user.pop("hashed_password")
         return user
 
+    def get_user_id(self, user_email):
```

Comment:
Shall we name this `get_user_by_email`? And return the entire `User` object instead.

Also, please validate the inputs of this with the schema